src/Agent.Plugins/Agent.Plugins.csproj

```diff
@@ -12,6 +12,7 @@
 
   <ItemGroup>
     <ProjectReference Include="..\Agent.Sdk\Agent.Sdk.csproj" />
+    <ProjectReference Include="..\Microsoft.VisualStudio.Services.Agent\Microsoft.VisualStudio.Services.Agent.csproj" />
```

Comment:
Can you make it an absolute reference.

Reply:
I would leave it. in ADO we should use absolute refs. But it seems OK in the agent side because there are a lot less projects to manage making path refactoring easy:
https://github.com/microsoft/azure-pipelines-agent/blob/ffb3a9b3e2eb5a1f34a0f45d0f2b8639740d37d3/src/Agent.Worker/Agent.Worker.csproj#L13

Reply:
I would also just leave it as it is since using relative path is consistent across agent repo.